README.md

```diff
@@ -1,68 +1,109 @@
+## Clone
 
-The build is minified and the filenames include the hashes.<br>
-Your app is ready to be deployed!
+---
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+- Clone this repo to you local machine using:
 
-### `npm run eject`
+  https://github.com/bsansone/campus-gaming-network.git
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+## Setup
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+---
 
-Instead, it will copy all the configuration files and the transitive dependencies (Webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+Once you have the project on your local machine, open up a terminal in the project folder and run,
 
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+```shell
+$ npm install
+```
 
-## Learn More
+to download and install the dependencies.
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+### Firebase
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+---
 
-### Code Splitting
+A Firebase project must be created to fully test and run the app. You can sign up using your google account here:
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/code-splitting
+https://firebase.google.com/
 
-### Analyzing the Bundle Size
+Create a project and call it `campus-gaming-network-test` or something along those lines. You will need to initialize a new database as well as enable the `Email/Password` authentication method. You can view the Firebase documentation here:
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size
+https://firebase.google.com/docs
 
-### Making a Progressive Web App
+To register the app to your firebase account you must:
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app
+1. Copy the contents of `.env.sample` into a new file called `.env.local`
 
-### Advanced Configuration
+2. Populate the firebase section of that `.env.local` file with your unique Firebase SDK snippet found under your firebase project settings.
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/advanced-configuration
+```javascript
+const firebaseConfig = {
+  apiKey: 'AIzaSyDOCAbC123dEf456GhI789jKl01-MnO',
```

Comment:
Are these your personal config values? We shouldn't put these in source control because it's open for anyone to see and they can access your database that way. I would just replace these with placeholder values like in `.env.sample`

Reply:
No its not mine I got it off the Firebase "getting started" section. I'll change it though.